telegram/ext/filters.py

```diff
@@ -1338,6 +1339,19 @@ def filter(self, message: Message) -> bool:
     """Use as ``filters.Document.ZIP``."""
 
 
+class _EffectId(MessageFilter):
+    __slots__ = ()
+
+    def filter(self, message: Message) -> bool:
+        return bool(message.effect_id)
+
+
+EFFECT_ID = _EffectId(name="filters.EFFECT_ID")
```

Comment:
Just an idea: allow filtering for specific effects? Dont strictly need that, also not in this PR, just mentioning

Reply:
the id is probably a long string so I don't think it makes much sense?